caiman/source_extraction/cnmf/estimates.py

```diff
@@ -1154,6 +1154,104 @@ def masks_2_neurofinder(self, dataset_name):
         bin_masks = self.A_thr.reshape([self.dims[0], self.dims[1], -1], order='F').transpose([2, 0, 1])
         return nf_masks_to_neurof_dict(bin_masks, dataset_name)
 
+    def save(self,
+             filename,
+             imaging_plane_name = None,
+             imaging_series_name = None,
+             sess_desc='CaImAn Results',
+             exp_desc=None,
+             imaging_rate=30,
+             location='somewhere in the brain',
+             orig_file_format='tiff'):
+        """save object in hdf5 file format
+
+        Args:
+            filename: str
+                path to the hdf5 file containing the saved object
+        """
+    #%%
+        from pynwb import NWBHDF5IO
```

Comment:
Since we're using conda, do imports inside the function make sense? Might be better to import at the top of the file unless we have a really good reason not to

Reply:
pywnb is not a dependency and I don't want to make it one right now. We could keep the import inside a function if that means that it would not try to execute unless we call the function. Is this the case?

Reply:
It was added to environment.yml so it was made into a dependency.